### PR TITLE
Update test blockchain - Closes #1124

### LINKED ISSUE #1124: Update test blockchain
### Description

For the CI to be run successfully, the test blockchain should be updated to the latest version of Lisk Core and the blockchain snapshot should contain all types of supported transactions.

### Acceptance Criteria

- Lisk Core image is updated to the latest version (Lisk Core v4)
- The snapshot cover all types of transactions supported by Lisk Core v4

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #1124 

### How was it solved?

- [x] The snapshot cover all types of transactions supported by Lisk Core v4
- [x] Update Jenkinsfile to import snapshots
- [x] Fix multisignature transaction indexing
- [x] Fix updateGeneratorTransaction event indexing issue
- [x] Update transaction execution status logic

### How was it tested?
Local + Jenkins

### CODE REVIEW

File: jenkins/Jenkinsfile.nightly [hunk 1]
```diff
@@ -52,7 +52,7 @@ pipeline {
 			when {
 				anyOf {
 					branch 'development'
-					branch 'feature/blockchain-agnostic'
+					branch 'feature/1157-prepare-lisk-service-v070-for-beta'
```
Comment (sameersubudhi) [inline jenkins/Jenkinsfile.nightly:55]: Is it possible to use a wildcard to run the jobs against any feature branch? Something like below:
Please check and apply the same below as well.
```suggestion
					branch 'feature/**'
```

File: services/blockchain-indexer/shared/indexer/transactionProcessor/auth/registerMultisignatureGroup.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ const multisignatureIndexSchema = require('../../../database/schema/multisignatu
 const getMultisignatureIndex = () => getTableInstance('multisignature', multisignatureIndexSchema, MYSQL_ENDPOINT);
 
 // Command specific constants
-const commandName = 'registerMultisignatureGroup';
+const commandName = 'registerMultisignature';
```
Comment (sameersubudhi) [inline services/blockchain-indexer/shared/indexer/transactionProcessor/auth/registerMultisignatureGroup.js:34]: Can we please rename the files to match the command name as well?

File: services/blockchain-connector/shared/sdk/decoder.js [hunk 3]
```diff
@@ -83,14 +83,18 @@ const decodeTransaction = (transaction) => {
...
+					Object.entries(entry).forEach(([innerKey, innerValue]) => {
+						if (innerKey.toLowerCase().endsWith('address') && !innerKey.includes('legacy')) {
+							formattedEntry[innerKey] = getLisk32Address(innerValue.toString('hex'));
+						} else {
+							formattedEntry[innerKey] = innerValue;
+						}
+					});
+				} else {
+					formattedEntry = entry;
+				}
```
Comment (sameersubudhi) [inline services/blockchain-connector/shared/sdk/decoder.js:97]:
```suggestion
				if (Buffer.isBuffer(entry)) {
					formattedEntry = entry;
				} else {
					Object.entries(entry).forEach(([innerKey, innerValue]) => {
						if (innerKey.toLowerCase().endsWith('address') && !innerKey.includes('legacy')) {
							formattedEntry[innerKey] = getLisk32Address(innerValue.toString('hex'));
						} else {
							formattedEntry[innerKey] = innerValue;
						}
					});
				}
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: jenkins/Jenkinsfile.nightly (hunk 1 above)
Comment (nagdahimanshu) [reply jenkins/Jenkinsfile.nightly:55]: I will check if possible and update the file.

**Review by Nazgolze — APPROVED** [review]
the Jenkinsfile changes look good

**Review by priojeetpriyom — APPROVED** [review]

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**